Sources/WebSocketKit/WebSocket.swift

```diff
@@ -4,21 +4,24 @@ import NIOHTTP1
 import NIOSSL
 import Foundation
 import NIOFoundationCompat
+import CompressNIO
 import NIOConcurrencyHelpers
 
 public final class WebSocket: Sendable {
-    enum PeerType: Sendable {
+    
+    public enum PeerType: Sendable {
```

Comment:
This doesn't need to be public

Reply:
I think it's actually needed to be public but I'll check once more. Seems it wa a couple days ago that I changed it so might not have the same api.

Reply:
I just checked and it needs to be public because types outside Websocket need it.
internal doesnt suffice because an inter needs it. I really think this should be like surface protein for a websocket. It's a perfectly fine enum and is useful for the classes around websocket to support its use.